wallet-new/README.md

```diff
@@ -1,26 +1,49 @@
 # Cardano Wallet API
 
-May this be the beginning of a beautiful program.
```

Comment:
:D

Reply:
For the nostalgic, this was the sentence that [SLIME](https://github.com/slime/slime) showed upon opening the REPL :)